tests/hyperv_test.go

```diff
@@ -139,7 +139,9 @@ var _ = Describe("[Serial][sig-compute] Hyper-V enlightenments", Serial, func()
 
 				for _, node := range nodeList.Items {
 					_, isNodeLabellerStopped := node.Annotations[v1.LabellerSkipNodeAnnotation]
-					Expect(isNodeLabellerStopped).To(BeTrue())
+					if !isNodeLabellerStopped {
```

Comment:
I would just simplify, `resumeNodeLabeller` already takes care of all of this logic (resuming the node-labeller if stopped and checking that the annotation disappeared), this is just a duplication of logic that caused the issue in the first place.

```go
AfterEach(func() {
	nodeList, err := virtClient.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
	Expect(err).ToNot(HaveOccurred())

	for _, node := range nodeList.Items {
		_ = resumeNodeLabeller(node.Name, virtClient)
	}
})
```

Reply:
Yeah, you're absolutely right. Thanks!
Done.